python/doc/index.rst

```diff
@@ -33,6 +33,17 @@ It offers the ability to:
   - extract marginal distributions of aGrUM Bayesian networks as OpenTURNS distributions
   - define and estimate bayesian networks parameterized by local conditional copula functions for each node (CBN)
 
+How to install it ?
+-------------------
+
+Using conda, you need to run the following command:
+
+.. code-block:: console
+
+   $ conda install -c conda-forge otagrum
```

Comment:
it's a bit more involved as conda-forge has to be the top priority channel and also it will be impossible from the anaconda installer, so maybe only provide the link

Reply:
My only problem with the link (which is the one given on the github repository of otagrum) is that it does not really say how to install otagrum since it doesn't say for example that you need to install aGrUM and OpenTURNS first if you do it from sources. I will give the details to do it but for someone who has never done such an installation it is quite hard. Isn't it possible to keep the conda command if I describe first how to do it from sources ? Because at the moment I'm afraid that only a few people can use otagrum.

Reply:
then at least add  "conda config --add channels conda-forge"
as conda-forge must always be top-priority for openturns